Description: Drop libbacktrace feature for now.
 We really don't want a trip through NEW right now.
Author: Peter Michael Green <plugwash@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-04-12

--- rust-backtrace-0.3.46.orig/Cargo.toml
+++ rust-backtrace-0.3.46/Cargo.toml
@@ -45,10 +45,10 @@ name = "smoke"
 required-features = ["std"]
 edition = "2018"
 
-[[test]]
-name = "accuracy"
-required-features = ["std", "dbghelp", "libbacktrace", "libunwind"]
-edition = "2018"
+#[[test]]
+#name = "accuracy"
+#required-features = ["std", "dbghelp", "libbacktrace", "libunwind"]
+#edition = "2018"
 
 [[test]]
 name = "concurrent-panics"
@@ -115,11 +115,11 @@ optional = true
 [features]
 coresymbolication = []
 dbghelp = []
-default = ["std", "libunwind", "libbacktrace", "dladdr", "dbghelp"]
+default = ["std", "libunwind", "dladdr", "dbghelp"]
 dladdr = []
 gimli-symbolize = ["addr2line", "findshlibs", "memmap", "goblin"]
 kernel32 = []
-libbacktrace = ["backtrace-sys/backtrace-sys"]
+#libbacktrace = ["backtrace-sys/backtrace-sys"]
 libunwind = []
 rustc-dep-of-std = ["backtrace-sys/rustc-dep-of-std", "cfg-if/rustc-dep-of-std", "core", "compiler_builtins", "libc/rustc-dep-of-std", "rustc-demangle/rustc-dep-of-std"]
 serialize-rustc = ["rustc-serialize"]
